### DISCUSSION TITLE
ESP32 verbindet sich nicht mit Fritzbox

### DISCUSSION THREAD

**Discussion by Melvin6328** [jomjol/AI-on-the-edge-device#2709] [Q&A]
Hallo,

ich habe gestern versucht meinen ESP32 einzurichten. Es hat alles funktioniert nun wollte ich in den Keller um dort den kleinen an den Stromzähler zu installieren. Kaum gehe ich runter verliert er das WLAN Signal alles schön und gut dachte ich mir denn unten im Keller steht eine FritzBox 7590 die mit der selben SSID und dem selben Passwort arbeitet wie die Repeater von DECO oben. Ich sehe nur die Fehlermeldung (201 AP). Ich habe es schon versucht mit die SSID von der FirtzBox zu ändern und eine feste IP aber nichts funktioniert. Habe ihn auch mehrmals neu geflasht und die SD Karte getauscht aber nichts funktioniert. Die FritzBox steht auch nur 1,5 Meter weit weg von dem ESP. Hat jemand eine Idee an was das liegen könnte?

**Comment by caco3** [comment]
Kannst Du mal versuchen, [rssithreshold](https://jomjol.github.io/AI-on-the-edge-device-docs/Parameters/#parameter-rssithreshold) zu aktivieren?

Ansonsten bitte mal ein Debug Log mit Aufstarten am Anfang.

**Reply by Melvin6328** [reply]
Danke für die Hilfe aber der Fehler liegt leider an der fritzbox die hat irgendeinen defekt sie zeigt das WLAN an aber hat dann keins sobald man sich einloggt die Repeater schon.
